frontend/src/util/tinymceInit.js

```diff
@@ -109,7 +111,7 @@ import { v4 as uniqueId } from 'uuid'
         'alignleft aligncenter alignright alignjustify | numlist bullist outdent indent | table | code codesample | insert | customFullscreen'
       ],
       codesample_global_prismjs: true,
-      codesample_languages: codeSampleLangList,
+      codesample_languages: reduxStore.system.config.ui__notes__code_sample_languages,
```

Comment:
as seen together, this variable might not be initialized.
wrap it around an if and display a console.error containing description of error and pointing to the issue related to this dev. And add a comment in the issue explaining that the solution would be to add an await to this.loadAppConfig() in Tracim.jsx